main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -2213,6 +2272,11 @@ public static int getAllHistoryCachesCount() {
             selectionArgs = new String[] { String.valueOf(cacheType.id) };
         }
 
+        selection.append(" and geocode in (SELECT geocode FROM " + dbTableCachesLists + " WHERE list_id ");
```

Comment:
just wondering why none of our static code analysis tools complains here (doing string concatenation inside of stringbuilder append calls is not optimal for performance)